dyc/base.py

```diff
@@ -25,7 +25,8 @@ def initialize(self, change=None):
             filename = fileinput.filename()
             lineno = fileinput.lineno()
             keywords = self.config.get('keywords')
-            found = (
+            found = false if line.lstrip()[0] in ("#", '"')
```

Comment:
I think we are basing found based on the initial comments Like
```python
"""Test"""
# Test
```

If so, these are defined in `config/defaults.yml` and we can change the docstrings to be `''''` instead of `"""`

Reply:
@Zarad1993 i couldnt completely understand , are you requesting to refer them from config/default.yml?

Reply:
Yes please.

Reply:
Tests are failing too.